aspnet/client-side/yeoman.rst

```diff
@@ -75,16 +75,11 @@ At this point you can navigate to the URL shown to test the newly created ASP.NE
 
 .. tip:: If you were directed to this tutorial from :doc:`/tutorials/your-first-mac-aspnet`, you can return now.
```

Comment:
I think we can now remove this tip since the Yeoman instructions are now embedded into the Mac tutorial.